app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show; end
+end
```

Comment:
Has users_controller and posts_controller with #index and #show actions 🐐 Magnific work!